recipe/meta.yaml

```diff
@@ -11,7 +12,13 @@ source:
   sha256: 724f3b430dca94733aea8325cdf5a9a2043184c3bc7dff8cf6452e1848efd4e3
 
 build:
-  number: 0
+  run_exports:
+    - {{ pin_subpackage( name|lower, max_pin='x.x') }}
```

Comment:
not sure if we need that.  I think it has to do with exporting shared libs, which STIR currently doesn' t do